nucliadb/nucliadb/ingest/fields/base.py

```diff
@@ -301,6 +303,16 @@ async def get_vectors(self, force=False) -> Optional[VectorObject]:
                 self.extracted_vectors = payload
         return self.extracted_vectors
 
+    async def get_user_vectors(self, force=False) -> Optional[UserVectors]:
+        if self.extracted_vectors is None or force:
```

Comment:
self.extracted_vectors is already being used to store the "regular vectors". Is it safe to reuse the attribute?